2017/proposals/bansal-utkarsh.md

```diff
@@ -0,0 +1,628 @@
+Port testsuite to pytest
+========================
+
+
+Abstract
+========
+Testing is crucial to the development of any software and MDAnalysis currently uses nose to test their code. 
+Unfortunately, [nose](http://nose.readthedocs.io/en/latest/) is no longer under active development so the community has decided to shift over to [pytest](http://doc.pytest.org/en/latest/).
+The objective of this project is to implement this shift in a way that existing development work is not affected
+and to standardize and improve the existing test cases.
+
+
+Current State
+=============
+Currently the testsuite consists of 8 modules that have a total of 4731 test cases. Running this testsuite with 
+pytest (without any modifications) results in the following output:
+
+`1789 failed, 2630 passed, 52 skipped, 1610 pytest-warnings, 26 error`
+
+Also, it is to be noted that pytest misses out some (231 to be exact) test cases too. This is due to differences 
+between test discovery rules of nose and pytest.
```

Comment:
Did you check the coverage number of pytest? It would also be nice to know if coverage drops with pytest currently.